systemtest/src/test/java/io/strimzi/systemtest/AllNamespaceST.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest;
+
+import io.fabric8.kubernetes.api.model.rbac.KubernetesClusterRoleBinding;
+import io.strimzi.test.annotations.ClusterOperator;
+import io.strimzi.test.annotations.Namespace;
+import io.strimzi.test.extensions.StrimziExtension;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.util.List;
+
+import static io.strimzi.test.extensions.StrimziExtension.REGRESSION;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.not;
+
+@ExtendWith(StrimziExtension.class)
+@Namespace(AllNamespaceST.CO_NAMESPACE)
+@Namespace(value = AllNamespaceST.SECOND_NAMESPACE, use = false)
+@Namespace(value = AllNamespaceST.THIRD_NAMESPACE, use = false)
+@ClusterOperator
+class AllNamespaceST extends AbstractNamespaceST {
+
+    private static final Logger LOGGER = LogManager.getLogger(AllNamespaceST.class);
+    static final String THIRD_NAMESPACE = "third-namespace-test";
+    private static Resources thirdNamespaceResources;
+
+    /**
+     * Test the case where the TO is configured to watch a different namespace that it is deployed in
+     */
+    @Test
+    @Tag(REGRESSION)
+    void testTopicOperatorWatchingOtherNamespace() {
```

Comment:
Do we need similar test for:
* User Operator?
* Standalone topic operator? (maybe not as critical since its deprecated feature)

Reply:
yes, it make sense to create coverage here for all components with different combinations.

Reply:
Added test cases for KafkaUser and Kafka Connect. As for standalone UO, I guess we no need to spend our time for the deprecated feature, because we have a long backlog with uncovered features, so priority for standalone UO, TO is really low.

Reply:
Sorry, what I meant with _standalone topic operator_ was the `topicOperator` object in the Kafka CR not the actual separate deployment. But I'm fine with it being low prio.